docs/Protocol/Protocol_v2/relay.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 2
+---
+
+# Relayers
+
+## Recap
+
+Powerloom Protocol V1 that powered the Incentivized testnets and the Lite Node testnet were able to submit their snapshots to the protocol state in the following manner: 
+
+1. The nodes submit their snapshots over HTTP to the relayers as EIP-712 signed message objects
+2. The relayers submit each node's submission against each project ID as [individual smart contract transactions to the method `submitSnapshot()`](/docs/Protocol/Specifications/protocol-state.md#function-submitsnapshotstring-memory-snapshotcid-uint256-epochid-string-memory-projectid-request-calldata-request-bytes-calldata-signature-public)
+
+## What's new in V2
+
+V2 relayers make use of the circuit relay transport protocol as implemented by the `libp2p` project. Briefly stated, this has two major advantages:
+
+* Faster, lightweight connections 
+  * Avoids overheads associated with HTTP connections: connection management, establishment, pooling, message payload size
+* More geographically available and censorship resistant
+
+To quote from the docs on circuit relays,
+
+```
+When a peer isn’t able to listen on a public address, it can dial out to a relay peer,
+which will keep a long-lived connection open. 
+
+Other peers will be able to dial through the relay peer using a 
+`p2p-circuit` address, which will forward traffic to its destination.
+```
+
+:::info
+[libp2p docs: Circuit Relay](https://docs.libp2p.io/concepts/nat/circuit-relay/)
+:::
+
```

Comment:
@anomit should we include trusted relayers here in this section now or later - this feature is currently in internal testing.